client.go

```diff
@@ -73,3 +79,12 @@ type Port struct {
 	Type   PortType
 	Name   string
 }
+
+// A DpipeTable is a devlink DPIPE table
```

Comment:
```suggestion
// A DPIPETable is a devlink pipeline debugging (DPIPE) table.
// For more information on DPIPE, see: https://github.com/Mellanox/mlxsw/wiki/Pipeline-Debugging-(DPIPE).
```

Reply:
This provides more doc clarification. What do you think?

Reply:
Fixed.